spec/reporter-spec.js

```diff
@@ -169,6 +169,16 @@ describe("Reporter", () => {
         {name: 'bundled-2', path: '/Applications/Atom.app/Contents/Resources/app.asar/node_modules/bundled-2', metadata: {version: '1.2.0'}},
       ]
 
+      const bundledPackages = [
+        'bundled-1',
+        'bundled-2'
+      ]
+
+      const packageDirPaths = ['/Users/user/.atom/packages']
+
+      spyOn(atom.packages, 'isBundledPackage').andCallFake((name) => bundledPackages.includes(name))
+      spyOn(atom.packages, 'getPackageDirPaths').andCallFake(() => packageDirPaths)
```

Comment:
`.andReturn(packageDirPaths)`

Reply:
Fixed in https://github.com/atom/exception-reporting/pull/37/commits/2992a390207f9001e3fcda10d57f03f6da0fb5df